### ISSUE TITLE
Improve SCons error reporting when building on macOS without the Vulkan SDK installed

### ISSUE THREAD

**Issue by ShlomiRex** [godotengine/godot#62914]
### Godot version

d26442e709f6361af9ac755ec9291bb43f2cd69b

### System information

macOS Monterey 12.4 Dual-core i3 Intel 

### Issue description

When trying to compile on my mac:

[The Issue](https://imgur.com/a/oiqjeX6)

I get the error:

```
shlomidomnenko@Shlomis-MacBook-Air godot % scons platform=osx arch=x86_64 target=release_debug -j2
scons: Reading SConscript files ...
Building for macOS 10.12+, platform x86_64.
FileNotFoundError: [Errno 2] No such file or directory: '/Users/shlomidomnenko/VulkanSDK':
  File "/Users/shlomidomnenko/Projects/godot/SConstruct", line 470:
    detect.configure(env)
  File "/Users/shlomidomnenko/Projects/godot/./platform/osx/detect.py", line 243:
    mvk_path = get_mvk_sdk_path()
  File "/Users/shlomidomnenko/Projects/godot/./platform/osx/detect.py", line 55:
    files = os.listdir(dirname)
```

### Steps to reproduce

1. Get macOS 12.4
2. Run the command `scons platform=osx arch=x86_64 target=release_debug -j2`
3. You get vulkan error

**Comment by fire** [comment]
The mac does not natively support vulkan. Please install the mac vulkan sdk. I don't have instructions on hand, but let me know if you have trouble.

**Comment by Calinou** [comment]
We should modify the SCons setup to print an error message in this case, and perhaps run https://github.com/godotengine/godot/pull/54555 automatically beforehand.

**Comment by ShlomiRex** [comment]
Vulkan can be found here: https://vulkan.lunarg.com/doc/sdk/1.3.216.0/mac/getting_started.html

After installing vulkan on all default settings, the compile works now. I'm building as we speak.

Default location is:
/Users/<user>/VulkanSDK

**Comment by Calinou** [comment]
> I'm working on the docs to show that Vulkan SDK is a requirement for macOS, spesifically version 4.

This is already mentioned in the `master` branch of the documentation (which maps to `/latest`): https://docs.godotengine.org/en/latest/development/compiling/compiling_for_macos.html

**Comment by yedpodtrzitko** [comment]
fix merged in #65024, can be closed now
